app/bundles/CoreBundle/Model/FormModel.php

```diff
@@ -50,10 +50,24 @@ public function isLocked($entity)
     {
         if (method_exists($entity, 'getCheckedOut')) {
             $checkedOut = $entity->getCheckedOut();
-            if (!empty($checkedOut)) {
-                //is it checked out by the current user?
+            if (!empty($checkedOut) && $checkedOut instanceof \DateTime) {
                 $checkedOutBy = $entity->getCheckedOutBy();
-                if (!empty($checkedOutBy) && $checkedOutBy !== $this->userHelper->getUser()->getId()) {
+                $maxLockTime  = $this->factory->getParameter('max_entity_lock_time', 0);
```

Comment:
The other way is call CoreParametersHelper contruction into `isLocked` function but it's not looking very efficient way.